bin/e2e.sh

```diff
@@ -21,16 +25,18 @@ done
 [[ ! -z "$hub" ]]       && args=$args" -h $hub"
 [[ ! -z "$namespace" ]] && args=$args" -n $namespace"
 
+args=$args" --mixerTag $mixerTag"
```

Comment:
can you process mixerTag flag like in line 11-15? bin/e2e takes same arguments as driver and passes them along.